zerver/lib/import_realm.py

```diff
@@ -179,16 +179,39 @@ def fix_upload_links(data: TableData, message_table: TableName) -> None:
     organization being imported (which is only determined at import
     time), we need to rewrite the URLs of links to uploaded files
     during the import process.
+
+    Applied to the attachments path_id found in messages of zerver_message and zerver_scheduledmessage tables.
     """
+
+    # No need to loop and check every message if there are no attachments in the first place.
```

Comment:
This comment should normally be inside the if statement, and also you should line-wrap your comments, like all the existing ones are.